tests/phpunit/tests/block-bindings/postMetaSource.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * Tests for Block Bindings API "core/post-meta" source.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.5.0
+ *
+ * @group blocks
+ * @group block-bindings
+ */
+class Tests_Block_Bindings_Post_Meta_Source extends WP_UnitTestCase {
+
+	/**
+	 * Sets up each test method.
+	 */
+	public function set_up() {
+		global $post;
+
+		parent::set_up();
+
+		$post = self::factory()->post->create_and_get( array() );
```

Comment:
The post can be created in `wpSetUpBeforeClass` to avoid repeating hits to the DB.

Reply:
Thanks for the tip! 🙂  I've changed it in [this commit](https://github.com/WordPress/wordpress-develop/pull/6197/commits/ce4aa0c78e953e4c2be1d285851f0585eda8e311).